README.md

```diff
@@ -57,6 +57,25 @@ vars:
     union_databases: ['shopify_usa','shopify_canada'] # use this if the data is in different databases/projects but uses the same schema name
 ```
 
+### Add Passthrough Columns
+This package includes all source columns defined in the [staging_columns.sql](https://github.com/fivetran/dbt_shopify_source/blob/master/macros/staging_columns.sql) macro. To add additional columns to this package, do so using our pass-through column variables. This is extremely useful if you'd like to include custom fields to the package.
+
+```yml
+# dbt_project.yml
+
+...
+config-version: 2
+
+vars:
+  shopify_source:
```

Comment:
the variables themselves are not referenced in the transform package, so we should be good to keep the source-package scope here